src/Dashboard.js

```diff
@@ -296,39 +296,52 @@ class Dashboard extends React.Component {
 
   createMapExplainer() {
     const header = "Observations";
-    const body = `Greenhouse gases are monitored by a network of sites across the city which provide us with near real-time data.
-    This includes both carbon dioxide and methane, the most influential greenhouse gases.`;
-    const explanation = `Start exploring the sites and the data by selecting a measurement site from the map`;
+    const body = `Greenhouse gas concentrations are monitored from a network of sites across the city.
+    Measurements are made of carbon dioxide and methane, the most important greenhouse gases.
+    Scientists are using these observations to learn more about the UK's methane emissions.`;
+    const explanation = `Start exploring the measurements by selecting a site from the map`;
     return <ExplanationBox header={header} intro={body} explain={explanation} />;
   }
 
   createEmissionsExplainer() {
     const header = "Emissions";
-    const intro = `On the live dashboard page we showed the amount of carbon dioxide and methane we measure at each location, but what we really want to know is where these greenhouse gases came from.`;
-    const body = `This is one way we can make sure we’re hitting planned targets.
-    We can build a map of expected emissions (an inventory) by adding together different sources.`;
+    const intro = `On the live dashboard page we showed the amount of carbon dioxide and methane we measure in the atmosphere.
+                  We make these measurements in order to infer emissions.`;
+    const body = `There are two primary methods for estimating greenhouse gas emissions:
+        a) Inventory methods, in which emissions are estimated using socioeconomic data (e.g., the amount of fuel sold and used in the UK).
+        A map showing the location of the UK's carbon dioxide and methane emissions, according to the inventory, is shown here.
+        b) Atmospheric data-based methods, in which concentration data and atmospheric models are compared to determine whether the inventory may need to be adjusted.`;
     return <ExplanationBox header={header} intro={intro} explain={body} />;
   }
 
   createIntro() {
-    const explanation = `Welcome to the OpenGHG dashboard where you can view live observation data from 
-                        our network of gas sensors across London.`;
+    const explanation = `Welcome to the OpenGHG dashboard, where you can view greenhouse gas concentration data from 
+                        our network sensors across London.`;
     return <ExplanationBox nogap={true} explain={explanation} />;
   }
 
+  createModelExplainer() {
```

Comment:
Before merging this would need to incorporate `createModelExplainer` into the `render()` function below, if we want this to display.